app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -151,9 +168,75 @@ fun EditActivityScreen(
               modifier = Modifier.padding(8.dp).fillMaxWidth().testTag("inputLocationEdit"),
               placeholder = { Text("Where is it taking place") },
           )
+          Column(
+              modifier = Modifier.fillMaxWidth().padding(8.dp).height(130.dp),
+              verticalArrangement = Arrangement.spacedBy(8.dp)) {
+                Button(
+                    onClick = { showDialog = true },
+                    modifier = Modifier.width(300.dp).height(40.dp).testTag("addAttendeeButton"),
+                ) {
+                  Row(
+                      horizontalArrangement =
+                          Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
+                      verticalAlignment = Alignment.CenterVertically,
+                  ) {
+                    Icon(
+                        Icons.Filled.Add,
+                        contentDescription = "add a new attendee",
+                    )
+                    Text("Add Attendee")
+                  }
+                }
+
+                LazyRow(
+                    modifier = Modifier.fillMaxHeight().height(85.dp).padding(8.dp),
+                ) {
+                  items(attendees.size) { index ->
```

Comment:
we should add the definition of what is an "attendee" in the wiki (in the conventions document)